packages/paste-website/gatsby-config.js

```diff
@@ -126,5 +126,17 @@ module.exports = {
       },
     },
     'gatsby-plugin-sitemap',
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: `Paste: A Design System for Twilio`,
```

Comment:
We should specify that it's an `internal` design system like we did elsewhere pre 0.1

Reply:
I added `internal` to constants and manifest.